### ISSUE TITLE
How to get the IRI of an entity in GraphQL-LD?

### ISSUE THREAD

**Issue by pheyvaer** [rubensworks/graphql-to-sparql.js#16]
If supported, what is the corresponding GraphQL-LD query of the following SPARQL querying?

 ```
SELECT ?s WHERE {
  ?s a <htttp://example.org/Test>.
}
```

I would think something like 
```
{ 
 ... on Test {}
}
```
but that doesn't work because there is nothing in the body.

**Comment by rubensworks** [comment]
This is indeed not possible at the moment.

I might add support for something like this:
```graphql
{ 
  _iri
  ... on Test {}
}
```

**Comment by JWerbrouck** [comment]
A possible solution for this could be this, with side effect that it selects all the variables in SPARQL:

JSON context:
`{
    "Test":         	"http://example.org/Test",
	"a":				"http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
}`

GraphQL query: 
`{ a(_:Test)}`

Returns the following: 
`SELECT * WHERE { ?b1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <htttp://example.org/Test>. }`

**Comment by rubensworks** [comment]
In the next release, this will be the way to do it (#15):

```graphql
{
  id
  a(_:Thing)
}
```

**Comment by rubensworks** [comment]
Released in 2.0.0

(FYI: https://github.com/rubensworks/GraphQL-LD.js)
